pkg/api/types.go

```diff
@@ -96,12 +97,11 @@ type APIVersions struct {
 
 // Scanner interface that all scanners should define.
 type Scanner interface {
-	// Scan will scan the image
-	Scan(string, *docker.Image) error
-	// ScannerName is the scanner's name
-	ScannerName() string
-	// ResultFileName returns the name of the results file
-	ResultsFileName() string
```

Comment:
@enoodle @pweil- also no need for these... the Scanner interface should just Scan :-)